pysilcam/silcamgui/interactive_summary.py

```diff
@@ -248,6 +250,42 @@ def __init__(self, parent=None):
         self.canvas.draw()
 
 
+    def old_data_check(self):
+        '''
+        checks fo HDF5 STATS and asks user to convert if needed.
+        will rename self.stats_filename for future use if needed.
+
+        returns bool which is True on success
+        '''
+        if not self.stats_filename.endswith('.csv'):
+            return True
+
+        h5_file = self.stats_filename.replace('.csv','.h5')
+        if os.path.isfile(h5_file):
+            self.stats_filename = h5_file
+            return True
+
+        msgBox = QMessageBox()
+        msgBox.setText('The STATS data appears to be out-dated csv file.' +
+                        '\n\nWe will now load the STATS file ' +
+                        'and convert the data to HDF5 files (which might take a while).')
+        msgBox.setIcon(QMessageBox.Question)
+        msgBox.setWindowTitle('Convert data?')
+        load_stats_button = msgBox.addButton('OK',
+                                                QMessageBox.ActionRole)
```

Comment:
E127: continuation line over-indented for visual indent  <!-- Automatically posted by Lintly -->